### PR TITLE
Create organizations list page #39

### PR DESCRIPTION
**Pull request by andpii**
Organizations Page example:
Old design:
image
New design:
image

### CODE REVIEW

**Comment by Setup1337** [comment]
image
Use our colorscheme for buttons please

**Comment by Setup1337** [comment]
Why second organization doesn`t have 'edit' button?

**Comment by andpii** [comment]
> Why second organization doesn`t have 'edit' button?

Because current user is not author of this organization. He cannot edit it. He is just a participant.

**Comment by andpii** [comment]
> image Use our colorscheme for buttons please

Done

File: frontend/src/app/components/organization-list/organization-list-item/organization-list-item.component.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  templateUrl: './organization-list-item.component.html',
+  styleUrls: ['./organization-list-item.component.sass']
+})
+export class OrganizationListItemComponent implements OnInit {
+  @Input() public currentUser: UserModel = {
+    name: 'Login1',
+    email: 'testlogin@gmail.com',
+    id: 1,
+    salt: 'Salt',
+    password: 'Password'
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/components/organization-list/organization-list-item/organization-list-item.component.ts]: Drop this fields in UserModel, please

Comment (andpii) [reply frontend/src/app/components/organization-list/organization-list-item/organization-list-item.component.ts]: Done

**Review by ismv — APPROVED** [review]

**Review by FeloniukM — APPROVED** [review]

**Pull request merged**